client/src/Components/common/ProductModal.jsx

```diff
@@ -15,22 +15,20 @@ function ProductModal({ open, onClose, product }) {
 
   const style = {
     position: 'absolute',
-    top: '50%',
-    left: '50%',
-    transform: 'translate(-50%, -50%)',
+    top: '20%',
+    left: '15%',
     width: '70%',
     height: '60%',
     bgcolor: 'background.paper',
     border: '2px solid #000',
     boxShadow: 24,
-    p: 4,
     borderRadius: 5,
-    padding: 10,
```

Comment:
1. I believe p and padding are identical?
2. Set the padding at the inner component